README.md

```diff
@@ -52,39 +52,47 @@ Deploy Hazelcast Enterprise Operator with the following command.
 
     oc apply -f operator.yaml
 
-#### Step 4: Start Hazelcast
+#### Step 4: Create Secret with Hazelcast License Key
 
-Add a Secret within the Project that contains the Hazelcast License Key. If you don't have one, get a trial key from this [link](https://hazelcast.com/hazelcast-enterprise-download/trial/).
+Base encode your Hazelcast License Key. If you don't have one, get a trial key from this [link](https://hazelcast.com/hazelcast-enterprise-download/trial/).
```

Comment:
What is Based encode your key? you mean base64?

Reply:
yes, changed, thanks for pointing it!